### DISCUSSION TITLE
Openwrt启动webdav失败

### DISCUSSION THREAD

**Discussion by lyh1980** [messense/aliyundrive-webdav#89] [Q&A] [answered]
相同的refresh-token在家里的OP启动没有问题，OP也能ping通aliyundrive.com
root@OpenWrt:~# aliyundrive-webdav --refresh-token fxxxxxcd4c654c1f850f976f0b6c7f9b
Oct 20 09:47:44.326 ERROR aliyundrive_webdav::drive: refresh token failed: HTTP status client error (400 Bad Request) for url (https://websv.aliyundrive.com/token/refresh): 
<h1>The input parameter refresh_token is not valid. Please refer to document. RequestId:BC9DFEEA-52B6-4E2B-9E94-E87A38BEC552</h1>
<h2>400</h2>
<h2>0FB71067-625C-4E30-BFDD-E0594931C01C</h2>

Error: initialize aliyundrive file system failed

**Comment by messense** [comment, marked as answer]
换一个新的 refresh token.
